src/Native/Runtime/amd64/InteropThunksHelpers.S

```diff
@@ -10,22 +10,22 @@
 LEAF_ENTRY RhpCommonStub, _TEXT
 
     PUSH_ARGUMENT_REGISTERS
+    push_register    r10
 
-    // +8 for alignment
-    alloc_stack    (SIZEOF_MAX_FP_ARG_SPILL + 8)
```

Comment:
Nit: I think that the original constant looked better than this shift. (Or you can call it `SIZEOF_FP_REGS` - it is used in other place in CoreRT to mean the same thing.)